.github/bench-dashboard/index.html

```
@@ -289,6 +289,10 @@ <h2 style="margin: 0 0 8px; font-size: 1.2rem">Level Profile (compress + decompr
           <label><input type="checkbox" data-series="ffi_decompress_speed" checked /> FFI decompress</label>
           <label><input type="checkbox" data-series="rust_ratio" checked /> Rust ratio</label>
           <label><input type="checkbox" data-series="ffi_ratio" checked /> FFI ratio</label>
+          <label><input type="checkbox" data-series="rust_compress_dict_speed" checked /> Rust compress-dict</label>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)